servers.json

```diff
@@ -1776,6 +1776,16 @@
         "minecraftVersions": ["1.8"],
         "primaryMinecraftVersion": "1.8"
     },
+    {
+        "id": "aquariusmc",
+        "name": "AquariusMC.it",
```

Comment:
```suggestion
        "name": "AquariusMC",
```

Reply:
We try not to use server addresses in general, could you please review my suggestion + commit if all okay?